src/era_5g_relay_network_application/era_5g_relay_network_application/client.py

```diff
@@ -95,14 +96,18 @@ def callback_image(data: Image, topic_name=None, topic_type=None):
         logger.error("You need to specify topic name and type!")
         return
     cv_image: Image = bridge.imgmsg_to_cv2(data, desired_encoding="bgr8")
-    if cv_image is not None:
-        client.send_image_ws(
-            cv_image,
-            Time.from_msg(data.header.stamp).nanoseconds,
-            metadata={"topic_name": topic_name, "topic_type": topic_type},  # TODO fix type annotation in send_image_ws?
-        )
-    else:
-        logger.warning("Empty image received!")
+    try:
```

Comment:
`try ... except` should ideally be only around `client.send_image_ws`. Not a big problem here, but in general, it is a good habit :)

Reply:
True, I changed it.